main_stylesheet.css

```diff
@@ -0,0 +1,127 @@
+
+* {
+    box-sizing: border-box;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:link, a:visited {
+    color: #CE3B45;
+}
+
+a:hover {
+    color: #215097;
+}
```

Comment:
- Can see some redundant selectors used like a for example where no a tags within php file. If you gonna be putting them in and styled ahead of time just ignore.

Reply:
yeah just left in for future links